ci-operator/config/openshift-eng/ocp-qe-perfscale-ci/openshift-eng-ocp-qe-perfscale-ci-main__aws-4.19-nightly-x86.yaml

```diff
@@ -311,8 +311,8 @@ tests:
       BASE_DOMAIN: perfscale.devcluster.openshift.com
       COMPUTE_NODE_TYPE: m5.xlarge
       EXTRA_FLAGS: --churn-duration=20m --service-latency
+      METRICS_PROFILE: metrics.yml,metrics-report.yml
```

Comment:
Let's change `metrics.yml` -> `metrics-aggregated.yml` OR delete this line entirely.